br/tests/br_autoid/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+#
+# Copyright 2023 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+DB="$TEST_NAME"
+
+run_sql "create database if not exists ${DB}"
+run_sql "create table $DB.issue46093 (a int primary key nonclustered auto_increment, b int) auto_id_cache = 1;"
+run_sql "insert into $DB.issue46093 (b) values (1), (2), (3);"
+run_sql "show table $DB.issue46093 next_row_id;"
+check_contains "NEXT_GLOBAL_ROW_ID: 30001"
+check_contains "NEXT_GLOBAL_ROW_ID: 4"
+
+run_sql "backup table $DB.issue46093 to 'local:///tmp/br'";
+run_sql "drop table $DB.issue46093;"
+run_sql "restore table $DB.issue46093 from 'local:///tmp/br';"
```

Comment:
please backup to `local://$TEST_DIR/$DB` to avoid duplicate backup paths in different tests.

Reply:
Done